pairing.go

```diff
@@ -134,3 +134,14 @@ func Pairing(p *G1Projective, q *G2Projective) *FQ12 {
 		{p.ToAffine(), G2AffineToPrepared(q.ToAffine())},
 	}))
 }
+
+// Compare e(P1, Q1) == e(P2, Q2)
```

Comment:
Can you make sure this matches godoc fomat starting with // CompareTwoPairings

Reply:
Is the patch https://github.com/phoreproject/bls/pull/14/commits/95f42e9e4146f9c085e1f123a0c4f843dc5f9c25 okay?

Reply:
Sorry, should be:

```go
// CompareTwoPairings checks e(P1, Q1) == e(P2, Q2)
// <=> FE(ML(P1, Q1)ML(-P2, Q2)) == 1
 func CompareTwoPairings(P1 *G1Projective, Q1 *G2Projective, P2 *G1Projective, Q2 *G2Projective) bool {
```

Then the linter will pass :]

Reply:
I'm sorry, I fixed it and force-pushed it. https://github.com/phoreproject/bls/compare/95f42e9e4146f9c085e1f123a0c4f843dc5f9c25..6b9dbe8c5372ea4e8359df85a9d48c47e2ded939